spec/predicates/human-review-crev.md

```diff
@@ -0,0 +1,123 @@
+# Predicate type: Dependency Reviews (crev)
+
+Type URI: (tentative) https://in-toto.io/attestation/human-review/crev/v0.1
+
+Version: Do we use Crev's versioning? Currently -1? What about their URI?
+
+## Purpose
+
+This attestation type is used to describe the results of human review of
+dependency source code. The format is based on the
+[crev project](https://github.com/crev-dev/crev).
+
+## Use Cases
+
+As noted above, crev enables social review of popular open source software
+dependencies. A crev review includes information such as the thoroughness of
+the review, understanding of the source code, and a final rating.
+
+## Prerequisite
+
+Understanding of the
+[in-toto attestation specification](https://github.com/in-toto/attestation).
```

Comment:
This isn't really a comment specific to this predicate, but I've seen several predicate specs say this, and I feel like this ought to just be implied. The pre-reqs I think are a lot more interesting and worth noting should be the lesser known, predicate-specific ones.